cfsctl/sync_command.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"log"
+
+	pb "github.com/c-fs/cfs/proto"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+)
+
+var (
+	syncName string
+)
+
+var syncCmd = &cobra.Command{
+	Use:   "sync",
+	Short: "sync files to cfs node",
+	Long:  "",
+	Run: func(cmd *cobra.Command, args []string) {
+		conn := setUpGrpcClient()
+		defer conn.Close()
+		c := pb.NewCfsClient(conn)
+
+		handleSync(context.TODO(), c)
+	},
+}
+
+func init() {
+	syncCmd.PersistentFlags().StringVarP(&syncName, "name", "n", "", "sync name")
```

Comment:
I don't think this will be an issue for now. Keep it this way unless it raise up as an issue.